CorsixTH/Lua/dialogs/resizables/menu_list_dialogs/custom_game.lua

```diff
@@ -97,6 +78,37 @@ function UICustomGame:UICustomGame(ui)
     :setTooltip(_S.tooltip.custom_game_window.load_selected_level)
 end
 
+--! Add a level to the Single Scenario list
+--!param level (string) the name of the level file
+--!param subdir (string) name of subdirectory the file resides under, otherwise use ""
+function UICustomGame:_addLevel(level, subdir)
+  local level_info = TheApp:readLevelFile(subdir .. level)
+  if level_info.name and level_info.map_file then
+    self.items[#self.items + 1] = {
+      name = level_info.name,
+      tooltip = _S.tooltip.custom_game_window.choose_game,
+      map_file = subdir .. level_info.map_file,
+      level_file = subdir .. level,
+      intro = level_info.briefing,
+      deprecated_variable_used = level_info.deprecated_variable_used,
+    }
+  end
+end
+
+--! Compile metainfo for all of the levels in the given path. For better Levels
+--! folder organisation, allow subdirectories one level deep
+--!param dir (string) The top-level directory that should contain level files.
+function UICustomGame:_findLevels(dir)
```

Comment:
This change allows levels one folder deep. That's what my comment was about, I wouldn't do that without considering the implications in it's own PR

Reply:
Ok makes sense now.